code/procs/scorestats.dm

```diff
@@ -352,6 +355,22 @@ var/datum/score_tracker/score_tracker
 
 		return jointext(., "")
 
+	proc/get_inspector_report()
+		var/report = ""
+		//this might be a really stupid way of finding all players of a specific job, please tell me if it is
+		for (var/mob/living/carbon/human/player in mobs)
+			if (player.job != "Inspector")
+				continue
+			report += "<B>Inspector [player.real_name]'s report</B><BR><HR>"
+			//find all clipboards they have on them (probably just one, but may as well check)
+			var/list/items = player.get_all_items_on_mob()
+			for (var/obj/item/clipboard/clipboard in items)
+				for(var/obj/item/paper/paper in clipboard.contents)
+					if (paper.name != "paper")
+						report += "<B>[paper.name]</B>"
+					report += paper.info
+		return report
+
```

Comment:
Don't like this- inefficient to loop through all mobs (and then all items on the inspector), and creates the issue where dropping or losing the clipboard close to the end of the round means your report doesn't show up. Would rather you track the clipboards themselves- maybe create a `clipboard/inspector` subtype, track that (with `START_TRACKING`), then iterate through all inspector clipboards at the end of the round and add their inspection notes to the ticket report. Would ensure the notes show up even if the inspector leaves their clipboard somewhere, and much more efficient.

That implementation does mean the inspector can't replace their one Important clipboard if they lose it, but I don't really mind that- could just name it "Inspector's Clipboard" so they can distinguish it from others. Also allows people to gank the inspector's clipboard and pin silly things on it, which will then show up at the end of the round regardless of who's holding it.

Reply:
Have now implemented pretty much exactly this, thanks for the pointers!